tests/Mvc/Database/DataExporterNullCheckTest.php

```
@@ -0,0 +1,391 @@
+use Phinx\Config\Config;
+use Phinx\Db\Adapter\AdapterInterface;
+use Phinx\Db\Adapter\AdapterFactory;
+
+/**
+ * Test null check handling in DataExporter's getTableCreateStatement method
+ * Verifies fix for crash when fetchRow returns null/false
+ */
+class DataExporterNullCheckTest extends TestCase
+{
+	private $tempDir;
+	private $originalFactory;
```

PHPMD (2.15.0)

Avoid unused private fields such as '$originalFactory'. (undefined)

(UnusedPrivateField)